app/server/methods/homes.js

```diff
@@ -270,5 +279,54 @@ Meteor.methods({
     }
 
     return Homes.find(selector).map(home => home._id);
+  },
+
+  assignManager({ groupId, userId }) {
+    // Get current user ID
+    const currentUserId = Meteor.userId();
+
+    // Check if current user is Admin
+    const currentUserIsAdmin = Roles.userIsInRole(currentUserId, "admin");
+    if (!currentUserIsAdmin) throw new Meteor.Error(500, 'User does not have right to perform this action')
+    if (!groupId || !userId)
+      throw new Meteor.Error(500, "Group or user not specified!");
+
+    return Permissions.update({ groupId, userId }, { $set: { isManager: true } }, { upsert: true })
+
+  },
+
+  getCurrentManagers(groupId) {
+    const ManagerPermissionRecords = Permissions.find({ groupId, isManager: true })
+    if (!ManagerPermissionRecords) return ManagerPermissionRecords
+
+    else {
+      const userIds = ManagerPermissionRecords.map(permissionRecord=>permissionRecord.userId)
+      const userInformation = Meteor.users.find({ _id: {$in :userIds} })
+      if (!userInformation) return userInformation
+      else {
+        let userEmailAddresses = [];
+        userEmailAddresses = userInformation.map(userDetail=>{
+          return userDetail.emails[0].address          
+        })
+        return userEmailAddresses
+      }
+    }
+  },
+  isHomeManagedByUser({ userId, homeId }) {
+    const homeInformation = Homes.find({ _id: homeId }).fetch();
```

Comment:
Just call this `home`.  

You can use `findOne()` instead of `find().fetch()[0]`, for cases where you expect only one result.